docs/tutorials/customize_runtime.md

```diff
@@ -24,7 +24,7 @@ You need to create a new directory named `mmdet/core/optimizer`.
 And then implement the new optimizer in a file, e.g., in `mmdet/core/optimizer/my_optimizer.py`:
```

Comment:
Where do you think is better to add the file? If we add at `mmdet`, we assume the user installed `mmdet ` via git clone. This is the same if we add at `mmcv`. I would prefer to add this file `mmdet3d/core/optimizer/` which is the easiest to implement.

Reply:
should add to mmdet3d since mmdet is taken as a lib by default